stac_fastapi/pgstac/app.py

```diff
@@ -64,17 +71,29 @@
 else:
     items_get_request_model = ItemCollectionUri
 
+if "collection_search" in _enabled_extensions:
+    collection_search_extension = CollectionSearchExtension.from_extensions(
+        extensions=extensions
+    )
+    collections_get_request_model = collection_search_extension.GET
+else:
+    collection_search_extension = None
+    collections_get_request_model = EmptyRequest
+
```

Comment:
This may just be a _me_ thing, but I find it a bit hard to follow variables being created within `if ... else` scope.  Inverting these may help legibility.
```py
enabled_extensions = (
    os.getenv("ENABLED_EXTENSIONS").split(",")
    if "ENABLED_EXTENSIONS" in os.environ
    else extensions_map.keys() + ["collection_search"]
)
extensions = [
    extension for key, extension in extensions_map.items() if key in enabled_extensions
]

items_get_request_model = (
    create_request_model(
        model_name="ItemCollectionUri",
        base_model=ItemCollectionUri,
        mixins=[TokenPaginationExtension().GET],
        request_type="GET",
    )
    if any(isinstance(ext, TokenPaginationExtension) for ext in extensions)
    else ItemCollectionUri
)

collection_search_extension = (
    CollectionSearchExtension.from_extensions(extensions)
    if "collection_search" in enabled_extensions
    else None
)
collections_get_request_model = (
    collection_search_extension.GET if collection_search_extension else EmptyRequest
)
```